fv3core/stencils/remapping_part1.py

```diff
@@ -116,8 +117,16 @@ def copy_j_adjacent(pe2: FloatField):
 
 
 @gtstencil()
-def pn2_and_pk(pe2: FloatField, pn2: FloatField, pk: FloatField, akap: float):
+def pn2_and_pk(
```

Comment:
Do we want to rename this to something involving delp as well, since it's also setting delp now?

Reply:
sure, good idea